### ISSUE TITLE
Trailing comma in import statement is not flagged as syntax error

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#2130]
The python interpreter generates a syntax error when there is a hanging comma in a `from A import B,` statement and there is no surrounding parentheses around the imported symbol list.

```python
from typing import Generic, TypeVar, Union,
```

> SyntaxError: trailing comma not allowed without surrounding parentheses

Pyright doesn't currently flag this as an error.

**Comment by erictraut** [comment]
This will be included in the next release.

**Comment by erictraut** [comment]
This is now addressed in pyright 1.1.159, which I just published. It will also be included in the next release of pylance.
